protos/external/goval/api/signing.proto

```diff
@@ -2,10 +2,10 @@ syntax = "proto3";
 
 import "google/protobuf/timestamp.proto";
 
-import "api/client.proto";
+import "protos/external/goval/api/client.proto";
 
 package api;
```

Comment:
Is it the `go_package` causing the conflict of the protobuf `package`?

Reply:
nope, it's the path!!(!?) the protobuf for baffling reasons decided to use that instead of the `go_package` for building the registry.